src/WorkspaceDependencyViewExtension/WorkspaceDependencyViewExtension.cs

```diff
@@ -20,11 +20,8 @@ public class WorkspaceDependencyViewExtension : IViewExtension, ILogSource
         internal MenuItem workspaceReferencesMenuItem;
         private const String extensionName = "Workspace References";
 
-        internal WorkspaceDependencyView DependencyView
-        {
-            get;
-            set;
-        }
+        internal WorkspaceDependencyView DependencyView;
```

Comment:
I don't think it makes sense to turn this into a field - there are tests which access this property external to the class which is usually done with a property - field access across objects is discouraged -  and below the setter is called inside the loaded function:

https://github.com/DynamoDS/Dynamo/pull/10437/files#diff-5c424b646bae377139b0c20ee304d7edR92

I see the getter called inside this class as well.

The reason it's likely you do not see these properties executed during normal testing is that this is a ViewExtension which will only be loaded inside tests which actually create a full DynamoView.

Reply:
I already did the change mjkkirschner .